eng/common/TestResources/sub-config/AzureOpenAITestingSub.json

```diff
@@ -0,0 +1,10 @@
+{
+    "SubscriptionId": "e72e5254-f265-4e95-9bd2-9ee8e7329051",
+    "TenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+    "TestApplicationId": "f850650c-1fcf-4489-b46f-71af2e30d360",
+    "TestApplicationOid": "30511c9d-ba1a-4c7b-b422-5b543da11b3f",
+    "ProvisionerApplicationId": "f850650c-1fcf-4489-b46f-71af2e30d360",
+    "ProvisionerApplicationOid": "30511c9d-ba1a-4c7b-b422-5b543da11b3f",
+    "Environment": "AzureCloud",
+    "AzureSubscription": "Speech Services - DEV - SDK (rob)"
```

Comment:
Is this value interesting?

Reply:
Not sure if it's needed, was just following the instructions. @danieljurek ?

Reply:
After looking... it's not relevant. I updated the [guidance docs](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/1080/Secret-auth-migration). Thanks!

Reply:
Thanks for the quick answer, I removed it!